extensions/ql-vscode/src/view/common/CodePaths/CodePaths.tsx

```diff
@@ -37,13 +42,28 @@ export const CodePaths = ({
     filterTelemetryOnValue: filterIsOpenTelemetry,
   });
 
+  const [scrollPosition, setScrollPosition] = useState<
+    ScollPosition | undefined
+  >(undefined);
+  React.useEffect(() => {
```

Comment:
```suggestion
useEffect(() => {
```
and importing it from above, wdyt?

Reply:
just an extra consideration from an external point of view: is the `useEffect` what you really need? In general I would use it if I need to run it when refreshing a page for example but here in vscode I don't know if the same concept exists and triggers the same.
If that's not the case, wouldn't be more suitable to use `useCallback` and call it `onClick` from below?

Reply:
I believe `useEffect` is the right thing, since scroll position changes are a side effect of the component.